presto-product-tests/src/main/java/io/prestosql/tests/hive/acid/TestFlatTableConvertedToTransactionalTable.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive.acid;
+
+import io.prestosql.tempto.query.QueryResult;
+import io.prestosql.tests.hive.HiveProductTest;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.HIVE_TRANSACTIONAL;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+
+public class TestFlatTableConvertedToTransactionalTable
+        extends HiveProductTest
+{
+    @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL})
+    public void testReadingFullAcidConvertedTable()
```

Comment:
call it `testReadFullAcidWithOriginalFiles` and move to `TestHiveTransactionalTable`

add a variant with bucketed table

Reply:
fixed.